README.md

```diff
@@ -1,3 +1,7 @@
+[![License: MSLA](https://img.shields.io/badge/License-MSLA-blue.svg)](LICENSES/LicenseRef-MSLA.txt)
+![Build Status](https://github.com/MathisMARION/wisun-br-linux/actions/workflows/build.yaml/badge.svg?branch=wip/mm/ci)
```

Comment:
You can adapt this link :
`https://github.com/MathisMARION/wisun-br-linux/actions/workflows/build.yaml/badge.svg?branch=master`
or simple
`https://github.com/MathisMARION/wisun-br-linux/actions/workflows/build.yaml/badge.svg`